lib/resources/example_json/dstu2_examples_patient.rb

```diff
@@ -3,7 +3,7 @@ module Resources
 
     DSTU2_PATIENT_1 ||= {
       "resourceType": "Patient",
-      "id": "9827347",
+      "id": "4596007",
```

Comment:
This id no longer matches its entries' fullUrl

Also, the self link for our patient search example is a search by name=Carr but our examples have Dr. Who names (not Carr names).  Probably want to have the example search by something other than name